README.md

```diff
@@ -1,3 +1,17 @@
+[![Build Status](https://travis-ci.org/andela-foladipo/docs-cabinet-cp2.svg?branch=develop)](https://travis-ci.org/andela-foladipo/docs-cabinet-cp2)
+[![Code Climate](https://codeclimate.com/github/andela-foladipo/docs-cabinet-cp2//badges/gpa.svg)](https://codeclimate.com/github/andela-foladipo/docs-cabinet-cp2/)
+[![Coverage Status](https://coveralls.io/repos/github/andela-foladipo/docs-cabinet-cp2/badge.svg?branch=chore%2F147025811%2Fsetup-frontend-dev-environment)](https://coveralls.io/github/andela-foladipo/docs-cabinet-cp2?branch=chore%2F147025811%2Fsetup-frontend-dev-environment)
+
+
+## UPDATE2: The Frontend is coming...
+We've now implemented all the endpoints required for this app. What we're
+ working on now is a user interface that you'll love and find as easy to
+ use as possible...
+
+## UPDATE: The Backend is coming...
+We are making a lot of progress in setting up the backend of this project. So,
+we should soon have something for you to play with and hack. Stay tuned!
+
 # Welcome to Docs Cabinet
 This is a brand new project, and there are no features implemented yet.
  However, there are a lot of great things in the pipeline. Head over
```

Comment:
It would be great if your readme tells us more about what the project is about, how to run it, how to test it, any dependencies/frameworks the project uses or is built on...and other relevant stuff like that.